modules/outbound-domain-event-handler/src/application/handler.ts

```diff
@@ -0,0 +1,96 @@
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+ * Modusbox
+ - Vijay Kumar Guthi <vijaya.guthi@modusbox.com>
+ --------------
+ ******/
+
+'use strict';
+
+import { ILogger } from '@mojaloop/logging-bc-public-types-lib';
+import {
+    IRunHandler,
+    KafkaDomainEventConsumer,
+    KafkaCommandEventProducer,
+    IEventConsumer,
+    DomainEventMessage,
+    OutboundDomainEventMessageName,
+    IKafkaEventConsumerOptions,
+    IKafkaEventProducerOptions,
+} from '@mojaloop/sdk-scheme-adapter-private-shared-lib';
+import { IDomainEventHandlerOptions } from '../types';
+import { handleSDKOutboundBulkRequestReceived } from './handlers';
+
+export class OutboundEventHandler implements IRunHandler {
+    private _logger: ILogger;
+
+    private _consumer: IEventConsumer;
+
+    private _commandProducer: KafkaCommandEventProducer;
+
+    private _domainEventHandlerOptions: IDomainEventHandlerOptions;
+
+    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+    async start(appConfig: any, logger: ILogger): Promise<void> {
+        this._logger = logger;
+        this._logger.info('start');
+
+        const consumerOptions: IKafkaEventConsumerOptions = appConfig.get('KAFKA.DOMAIN_EVENT_CONSUMER');
+        this._consumer = new KafkaDomainEventConsumer(this._messageHandler.bind(this), consumerOptions, logger);
+        logger.info(`Created kafkaConsumer of type ${this._consumer.constructor.name}`);
+
+        /* eslint-disable-next-line @typescript-eslint/no-misused-promises */
+        await this._consumer.init();
+        await this._consumer.start();
+
+        const producerOptions: IKafkaEventProducerOptions = appConfig.get('KAFKA.COMMAND_EVENT_PRODUCER');
+        this._commandProducer = new KafkaCommandEventProducer(producerOptions, logger);
+        logger.info(`Created kafkaProducer of type ${this._commandProducer.constructor.name}`);
+        await this._commandProducer.init();
+
+        // Create options for handlers
+        this._domainEventHandlerOptions = {
+            commandProducer: this._commandProducer,
+        };
+    }
+
+    async destroy(): Promise<void> {
+        await Promise.all([
+            this._consumer?.destroy(),
+            this._commandProducer?.destroy(),
+        ]);
+    }
+
+    async _messageHandler(message: DomainEventMessage): Promise<void> {
+        this._logger.info(`Got domain event message: ${message.getName()}`);
+        console.log(message);
+        switch (message.getName()) {
+            case OutboundDomainEventMessageName.SDKOutboundBulkRequestReceived: {
```

Comment:
The same comment applies here as per the Cmd Handler.

We should be using the Message Entity Types themselves and not relying on an enum:

```suggestion
            case SDKOutboundBulkRequestReceived.name : {
```

Reply:
I removed this unnecessary file